src/App.tsx

```diff
@@ -1,14 +1,47 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visiblTodos, setVisiblTodos] = useState<Todo[]>([]);
+  const [errorLoad, setErrorLoad] = useState(false);
+  const [selectedAll, setSelectedAll] = useState(true);
+  const [selectedActive, setSelectedActive] = useState(false);
+  const [selectedCompleted, setSelectedCompleted] = useState(false);
+
+  const ItemLeft: number = todos.filter(
```

Comment:
```suggestion
  const itemLeft: number = todos.filter(
```
follow the camelCase